src/index.js

```diff
@@ -86,6 +87,7 @@ function isApplicationJson (headers: Headers): boolean {
 
 function assertExpectedStatus <T: {+status: number}> (expectedStatuses: ?Array<number>, jsonFetchResponse: T): void {
   if (Array.isArray(expectedStatuses) && expectedStatuses.indexOf(jsonFetchResponse.status) === -1) {
+  if (Array.isArray(expectedStatuses) && !_.includes(expectedStatuses, jsonFetchResponse.status)) {
```

Comment:
you can use native array `includes`, [supported since node 6](https://node.green/) which it looks like is the minimum we're supporting in this module